src/domain/transfer/index.js

```diff
@@ -59,6 +60,10 @@ const prepare = async (headers, message, dataUri) => {
       from: message.payerFsp,
       type: 'application/json',
       content: {
+        // TODO: will message.transferId always exist?
```

Comment:
On a prepare you should always have a message.transferId as it is required by Joi validation